src/Arcus.Security.Providers.HashiCorp/HashiCorpSecretProvider.cs

```diff
@@ -138,11 +138,11 @@ protected async Task<SecretData> GetTrackedSecretAsync(string secretName)
             };
 
             var isSuccessful = false;
-#if NET6_0
+//#if NET6_0 || NET8_0
             using (DurationMeasurement measurement = DurationMeasurement.Start())
-#else
-            using (DependencyMeasurement measurement = DependencyMeasurement.Start()) 
-#endif
+//#else
+//            using (DependencyMeasurement measurement = DependencyMeasurement.Start()) 
+//#endif
```

Comment:
commented code can be removed :)

Reply:
Oops, think I wasn't seeing clearly at the end. 😅 Thx!
Will fix these and the directives. 👍

Reply:
Removed comments here: https://github.com/arcus-azure/arcus.security/pull/421/commits/d24b66901b7efe25e896f007b5b6a323722a58c4